datamade/nwss-data-standard
# Handle ingested date formatting

## Overview

This PR removes transformation of ingested values and uses the provided `dateNF` option to specify how dates should be formatted.

## Testing Instructions

* Follow the test instructions [here](https://github.com/datamade/nwss-data-standard/pull/15) and confirm there are not date errors in the output. N.b., there will be time errors. Again, we _do not_ want to transform ingested values to pass validation. The data dictionary seems to expect `HH:MM`, while the JSON Schema date field seems to want the seconds, as well – I think it might be more straightforward to convert this field to a string and match against a pattern. Let's discuss / handle that issue separately.
